syntax = "proto3";

message ImageReferenceMsg {
    string session = 1;
    string image = 2;
}

message SessionReferenceMsg {
    string session = 1;
}

message SessionDetailsMsg {
    string session = 1;
    int32 images = 2;
}

message ImageSequenceHeaderMsg {
    string session = 1;
    int32 image = 2;
    int64 created = 3;
    int64 updated = 4;
    float score = 5;
    int32 clazz = 6;;
    int32 width = 7;
    int32 height = 8;;
    int32 segments = 9;
}

message ImageSequenceSegmentMsg {
    int32 segment = 1;
    bytes data = 2;
 }